.github/actions/prepare-compat-testing/action.yaml

```diff
@@ -0,0 +1,49 @@
+name: Prepare compatibility testing profiles
+
+inputs:
+  platform:
+    description: 'Which platform to download releases for?'
+    required: false
+    default: 'ubuntu22.04-amd64'
+    type: string
+
+outputs:
+  profiles-file:
+    description: 'Name of the file containing compatibility profiles'
+    value: ${{ steps.generate.outputs.file }}
+
+runs:
+  using: composite
+  steps:
+    - shell: bash
+      run: cat .env | tee -a $GITHUB_ENV
+    - name: Compute oldest compatible release version
+      id: vsn
+      shell: bash
+      run: |
+        prefix=$(echo $PROFILE | grep -q 'enterprise' && echo 'e' || echo 'v')
+        compat=$(./scripts/semver.sh ${PKG_VSN} | jq -r '"\(.major).\(.minor-1).0"')
```

Comment:
in 4.4, we have `relup-paths.eterm` and `relup-paths-ee.eterm` for hot-upgrade paths.
we can maybe add `rel/rolling-upgrade-paths.eterm` and `rel/rolling-upgrade-paths-ee.eterm` for `master` branch.

for opensource edition, we support upgrade from 5.0, 5.1 and 5.2 to `5.3.*` (latest, i.e. the current PR) 
for enterprise edition, we support upgrade from 5.1, 5.2 and 5.2 to `5.3.*` (latest, i.e. the current PR)

Reply:
then we can think about how to keep the paths in sync with upgrade paths which should be published to doc site.

Reply:
> we can maybe add `rel/rolling-upgrade-paths.eterm` and `rel/rolling-upgrade-paths-ee.eterm` for `master` branch.
> for opensource edition, we support upgrade from 5.0, 5.1 and 5.2 to 5.3.* (latest, i.e. the current PR)
> for enterprise edition, we support upgrade from 5.1, 5.2 and 5.2 to 5.3.* (latest, i.e. the current PR)

Thanks for the info, that's important. To clarify: the `rel/rolling-upgrade-paths.eterm` would then contain something along the lines of:
```
{<<"5.3.1">>,
 #{from_versions => [<<"5.3.0">>,<<"5.2.1">>,<<"5.2.0">>,<<"5.1.0">>,<<"5.0.0">>],
 ...
}}.
```

Looks correct? It would also imply the need to test with releases built with older Erlang/OTP. Currently, this PR is being kind of lazy in the sense that it's using the same Erlang/OTP runtime to run older code, and I wonder if this will become a problem at some point in the future?

> then we can think about how to keep the paths in sync with upgrade paths which should be published to doc site.

Also, kinda curious in what context this stuff should be published to the doc site. Can you elaborate a little bit on that?